com.jetbrains.WebStorm.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: com.jetbrains.WebStorm
+runtime: org.freedesktop.Sdk
+runtime-version: "18.08"
+sdk: org.freedesktop.Sdk
+command: webstorm-desktop
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=host
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --filesystem=xdg-run/keyring
+  - --device=dri
+modules:
+  - name: nodejs
```

Comment:
I'm not sure I understand what the problem is. I'm not too familiar with node's bits and bobs.

WebStorm integrates with both yarn and npm, you can install packages directly from the IDE. The project would use its own packages, not the systems, right?

You can also chose whichever version of node.js you want since it has `host` access, but I include the LTS in the package just in case the host hasn't got node installed.